charts/kube-state-metrics/templates/deployment.yaml

```diff
@@ -69,91 +69,91 @@ spec:
-        - --collectors=ingresses
+        - --resources=ingresses
 {{  end  }}
 {{  if .Values.collectors.jobs  }}
-        - --collectors=jobs
+        - --resources=jobs
 {{  end  }}
 {{  if .Values.collectors.limitranges  }}
-        - --collectors=limitranges
+        - --resources=limitranges
 {{  end  }}
 {{  if .Values.collectors.mutatingwebhookconfigurations  }}
-        - --collectors=mutatingwebhookconfigurations
+        - --resources=mutatingwebhookconfigurations
 {{  end  }}
 {{  if .Values.collectors.namespaces  }}
-        - --collectors=namespaces
+        - --resources=namespaces
 {{  end  }}
 {{  if .Values.collectors.networkpolicies  }}
-        - --collectors=networkpolicies
+        - --resources=networkpolicies
 {{  end  }}
 {{  if .Values.collectors.nodes  }}
-        - --collectors=nodes
+        - --resources=nodes
 {{  end  }}
 {{  if .Values.collectors.persistentvolumeclaims  }}
-        - --collectors=persistentvolumeclaims
+        - --resources=persistentvolumeclaims
 {{  end  }}
 {{  if .Values.collectors.persistentvolumes  }}
-        - --collectors=persistentvolumes
+        - --resources=persistentvolumes
 {{  end  }}
 {{  if .Values.collectors.poddisruptionbudgets  }}
-        - --collectors=poddisruptionbudgets
+        - --resources=poddisruptionbudgets
 {{  end  }}
 {{  if .Values.collectors.pods  }}
-        - --collectors=pods
+        - --resources=pods
 {{  end  }}
 {{  if .Values.collectors.replicasets  }}
-        - --collectors=replicasets
+        - --resources=replicasets
 {{  end  }}
 {{  if .Values.collectors.replicationcontrollers  }}
-        - --collectors=replicationcontrollers
+        - --resources=replicationcontrollers
 {{  end  }}
 {{  if .Values.collectors.resourcequotas  }}
-        - --collectors=resourcequotas
+        - --resources=resourcequotas
 {{  end  }}
 {{  if .Values.collectors.secrets  }}
-        - --collectors=secrets
+        - --resources=secrets
 {{  end  }}
 {{  if .Values.collectors.services  }}
-        - --collectors=services
+        - --resources=services
 {{  end  }}
 {{  if .Values.collectors.statefulsets  }}
-        - --collectors=statefulsets
+        - --resources=statefulsets
 {{  end  }}
 {{  if .Values.collectors.storageclasses  }}
-        - --collectors=storageclasses
+        - --resources=storageclasses
 {{  end  }}
 {{  if .Values.collectors.validatingwebhookconfigurations  }}
-        - --collectors=validatingwebhookconfigurations
+        - --resources=validatingwebhookconfigurations
 {{  end  }}
 {{  if .Values.collectors.verticalpodautoscalers  }}
-        - --collectors=verticalpodautoscalers
+        - --resources=verticalpodautoscalers
 {{  end  }}
 {{  if .Values.collectors.volumeattachments  }}
-        - --collectors=volumeattachments
+        - --resources=volumeattachments
 {{  end  }}
 {{ if .Values.namespace }}
-        - --namespace={{ .Values.namespace | join "," }}
+        - --namespaces={{ .Values.namespace | join "," }}
```

Comment:
Can we update the helm values `.Values.namespace` to `.Values.namespaces` to reflect this change?

Reply:
in L155 I mean

Reply:
Sure we can. I'll add a migration guide then.

Reply:
Yes! Thanks a lot :)

Reply:
Added a small upgrade guide and renamed this key.